Makefile.js

```diff
@@ -282,6 +282,9 @@ function generateRelease() {
     generateBlogPost(releaseInfo);
     commitSiteToGit(`v${releaseInfo.version}`);
 
+    echo("Generating eslintVersion.js for docs");
+    fs.writeFileSync("docs/src/_data/eslintVersion.js", `module.exports = "${releaseInfo.version}";`);
```

Comment:
In the current version, the eslint version can be passed in via environment variables. Maybe we should keep this feature.

ref https://github.com/eslint/eslint/blob/dcf178e3c41e7777ee76ae9582387be85118118b/docs/src/_data/eslintVersion.js#L19

Reply:
if line 286 was instead 
```
process.env.ESLINT_VERSION = releaseInfo.version
``` 
would that also lead to the version in the docs being the expected one? Sorry for the dumb question, I am not very clear on the `generate-release` script and how the docs are actually generated neither was I able to run it locally

Reply:
No. This is because environment variables are passed in when running the documentation, not when executing generate-release.

Reply:
I don’t want to read from the environment or anywhere else. The version should be static to the repo so we can easily copy files elsewhere and have it work correctly.

Reply:
Oops sorry, @kecrily is right. We still do need to read from env in order to support the HEAD branch.

Reply:
ok, so I will leave the `eslintVersion.js` file as it is and also generate a static one during the `generate-release` script as in the PR. Is there any other change you would suggest in this PR?

Reply:
After thinking about this some more, here's what I think will probably work best:

1. Update `eslintVersion.js` so that it reads from the `package.json` file in the `docs/` directory instead of the one in the root directory. Otherwise, it will remain the same.
2. In `Makefile.js` update the `version` field of `docs/package.json` to the newly-created release version.

Make sense?

Reply:
sounds good

Reply:
updated the pull request, if you could please review and provide your comments